packages/data/lib/di/injector.config.dart

```diff
@@ -0,0 +1,21 @@
+// // GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
wrote about it in another PR. in case of DI without codegen u shouldn't have this file at all